dvc/ignore_templates/__init__.py

```diff
@@ -0,0 +1,2 @@
+# templates for .dvcignore files
```

Comment:
This doesn't really have to be a module, as there is no actual code. You can either keep it as datafiles or maybe actually make these into the python shape by declaring these as strings. The latter way is easier to manage, as things like setup.py and pyinstaller will have a much easier time collecting these automatically.